### ISSUE TITLE
The Microsoft narrator focus does not sync with the keyboard focus on the menu item

### ISSUE THREAD

**Issue by bazzdg** [Microsoft/monaco-editor#971]
**Actual**:
While navigating the menu item of the explore button and press control+Shift+E, the keyboard focus moves in the menu item. The screen reader focus does not sync with the keyboard focus and screen reader does not announce the name of the files and solutions files name.

**Expected**:
The focus order on the page should be logical, it should move from left to right and the screen reader focus should sync with the keyboard focus and should announce all the file names.

**User Impact**:
Keyboard only and screen reader users will find difficulty if the screen reader focus does not sync with the keyboard focus.

**Test Environment**:
OS: Win 10
OS version: 1709 
OS build: 16299.98
Browser : Edge

MAS Reference:
MAS 4.2.10 - Tracking of focus and selection attribute

**Comment by alexandrudima** [comment]
@bazzdg What product are you testing? That is outside of the monaco editor.
